defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-postgresql_support_libpq_version: "9.5.*.pgdg14.04+2"
+postgresql_support_libpq_version: "9.5.*.pgdg*"
```

Comment:
I'm using that Ansible version as well. It may appear to work if the package is already installed, but if it actually has to install the package, it fails:

```
TASK [azavea.postgresql-support : Install client API libraries for PostgreSQL] *
fatal: [default]: FAILED! => {"cache_update_time": 0, "cache_updated": false, "changed": false, "failed": true, "msg": "'/usr/bin/apt-get -y -o \"Dpkg::Options::=--force-confdef\" -o \"Dpkg::Options::=--force-confold\"   install 'libpq-dev=9.5.*.pgdg*'' failed: E: Version '9.5.*.pgdg*' for 'libpq-dev' was not found\n", "stderr": "E: Version '9.5.*.pgdg*' for 'libpq-dev' was not found\n", "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\n", "stdout_lines": ["Reading package lists...", "Building dependency tree...", "Reading state information..."]}
```